cfgov/unprocessed/apps/research-reports/css/main.less

```diff
@@ -22,12 +22,12 @@
   height: 100%;
   & > ul {
     a {
-      padding-left: 0.75em;
+      padding-left: 1.85em;
+      text-indent: -1.1em;
     }
   }
-  .m-nav-link.current-section {
-    color: #101820;
-    border-color: #20aa3f;
+  h3, h4 {
+    padding-left: 19px;
```

Comment:
Should this be ems?

Reply:
They need to be the same so that everything is left aligned. I used the unit helper for this as well though